### PR TITLE
Feature/add financial component texts and styles

### PR DESCRIPTION
**Pull request by santiagonoguez**
I moved styles and texts to common header.

### CODE REVIEW

File: src/locales/locales_json/en/template.json [hunk 1]
```diff
@@ -7,7 +7,43 @@
...
+      "world-indexes": "Most Popular World Indexes"
+    },
+    "no-results": {
+      "title": "No Results Found",
+      "question": "Can't find what you're looking for?",
+      "verify-start": "Verify the instrument is valid",
+      "verify-mid": "or contact",
+      "verify-end": "for help."
+    },
+    "search-category": {
```
Comment (stulees) [inline src/locales/locales_json/en/template.json]: @santiagonoguez We don't have UI for searching category, is this necessary?

File: src/locales/locales_json/en/template.json [hunk 2]
```diff
@@ -7,7 +7,43 @@
     "delete-warning": "Are you sure you want to delete this Presentation?"
   },
   "financial": {
-    "currencies": "Currencies"
+    "add-category": {
```
Comment (stulees) [inline src/locales/locales_json/en/template.json]: @santiagonoguez We don't have UI for adding category, is this necessary?

File: src/locales/locales_json/en/template.json [hunk 3]
```diff
@@ -7,7 +7,43 @@
...
+      "currencies": "Add Currencies",
+      "market-statistics": "Add Market Statistics",
+      "stocks": "Add Stocks",
+      "world-indexes": "Add World Indexes"
+    },
+    "bonds": "Bonds",
+    "commodities": "Commodities",
+    "currencies": "Currencies",
+    "market-statistics": "Market Statistics",
+    "most-popular-category": {
```
Comment (stulees) [inline src/locales/locales_json/en/template.json]: @santiagonoguez Can you clarify where in our UI this is used? Thanks

File: src/locales/locales_json/en/template.json (hunk 1 above)
Comment (santiagonoguez) [reply src/locales/locales_json/en/template.json]: @stulees  There's a placeholder here that will need to be adjusted depending on the category:

https://github.com/Rise-Vision/rise-vision-apps/blob/feature/instrument-selector-navigation/web/partials/template-editor/components/component-financial.html#L170

File: src/locales/locales_json/en/template.json (hunk 2 above)
Comment (santiagonoguez) [reply src/locales/locales_json/en/template.json]: @stulees  We have a button here that needs to be adjusted depending on the category:

https://github.com/Rise-Vision/rise-vision-apps/blob/feature/instrument-selector-navigation/web/partials/template-editor/components/component-financial.html#L157

File: src/locales/locales_json/en/template.json (hunk 3 above)
Comment (santiagonoguez) [reply src/locales/locales_json/en/template.json]: @stulees this title needs to be updated according to the category:

https://github.com/Rise-Vision/rise-vision-apps/blob/feature/instrument-selector-navigation/web/partials/template-editor/components/component-financial.html#L190

File: src/locales/locales_json/en/template.json (hunk 1 above)
Comment (stulees) [reply src/locales/locales_json/en/template.json]: @santiagonoguez Got it, thanks

File: src/locales/locales_json/en/template.json (hunk 2 above)
Comment (stulees) [reply src/locales/locales_json/en/template.json]: @santiagonoguez Got it, thanks

File: src/locales/locales_json/en/template.json (hunk 3 above)
Comment (stulees) [reply src/locales/locales_json/en/template.json]: @santiagonoguez 👍

**Pull request merged**